### PR TITLE
Import add identifier in uri

### LINKED ISSUE #49: [Import] Importation des identifierInURI et parentClassOf et parentPropertyOf
Il faut pouvoir importer (dans le fichier xml d'import)

les balises identifierInURI, parentClassOf, parentPropertyOf

### PR DESCRIPTION
**Pull request by perrauda**
Resolves #49

### CODE REVIEW

**Review by MPica — APPROVED** [review]
All seems in order and to work as planned.

File: src/AppBundle/Controller/ProjectController.php [hunk 1]
```diff
@@ -369,46 +370,50 @@ public function editAction(Project $project, Request $request)
...
-                                    die;
+                            if(!is_null($nodeXmlClass->textProperties->scopeNote)){
+                                foreach($nodeXmlClass->textProperties->scopeNote as $keySn => $nodeXmlScopeNote) {
+                                    if(!$langCollection->contains((string)$nodeXmlScopeNote->attributes()->lang)){
+                                        $langCollection->add((string)$nodeXmlScopeNote->attributes()->lang);
+                                    }
+                                    else{
+                                        var_dump($langCollection);
+                                        echo (string)$nodeXmlScopeNote->attributes()->lang;
+                                        echo "- 2 scopes notes au moins ont la même langue. j".$newClassVersion->getClass()->getIdentifierInNamespace();
```
Comment (dferhod) [inline src/AppBundle/Controller/ProjectController.php:381]: corriger le "j" qui traine

File: src/AppBundle/Controller/ProjectController.php [hunk 2]
```diff
@@ -677,43 +673,47 @@ public function editAction(Project $project, Request $request)
...
-                                    echo "- 2 scopes notes au moins ont la même langue. j" . $newClassVersion->getClass()->getIdentifierInNamespace();
-                                    die;
+                            if (!is_null($nodeXmlProperty->textProperties->scopeNote)) {
+                                foreach ($nodeXmlProperty->textProperties->scopeNote as $keySn => $nodeXmlScopeNote) {
+                                    if (!$langCollection->contains((string)$nodeXmlScopeNote->attributes()->lang)) {
+                                        $langCollection->add((string)$nodeXmlScopeNote->attributes()->lang);
+                                    } else {
+                                        var_dump($langCollection);
+                                        echo (string)$nodeXmlScopeNote->attributes()->lang;
+                                        echo "- 2 scopes notes au moins ont la même langue. j" . $newClassVersion->getClass()->getIdentifierInNamespace();
```
Comment (dferhod) [inline src/AppBundle/Controller/ProjectController.php:683]: corriger le "j"

**Review by dferhod — CHANGES_REQUESTED** [review]
Voir commentaires, sinon c'est bon

**Pull request merged**